default.yml

```diff
@@ -2,108 +2,100 @@
 # https://rubocop.readthedocs.io
 
 AllCops:
-  DisplayCopNames: true
   Exclude:
     - '*.gemspec'
```

Comment:
We should include gemspecs IMHO, it will sort dependencies alphabetically and other nice things.